webpack/assets/javascripts/foreman_tools.js

```diff
@@ -90,6 +90,9 @@ export function updateTable(element) {
   if (element !== undefined) {
     search = $(element).find('.autocomplete-input').val() || $('#search-form').find('.autocomplete-input').val();
     if (search !== undefined) {
+      if (getQueryStringValue(uri, 'page') > 1) {
```

Comment:
I dont think thats correct, if i have a search term and then I'll do pagination, wont this always keep me on the first page?

IMHO only when the search has been submitted (not if it has a value) then you reset the page number to 1. (or remove the page attribute from the url)